From 8415a58cf5aa0e2abfb84a19f217ff576fb99c8b Mon Sep 17 00:00:00 2001
From: Orlando Vazquez <ovazquez@gmail.com>
Date: Thu, 21 Mar 2019 12:57:10 -0700
Subject: [PATCH] TRITON-1340 VMAPI flexible disks test times out at import
 step after 2 minutes Reviewed by: Trent Mick <trentm@gmail.com>

---
 package.json   | 2 +-
 test/common.js | 7 +++++--
 2 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/package.json b/package.json
index b6908c9..ccfa1ae 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.7.1",
+  "version": "9.7.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
diff --git a/test/common.js b/test/common.js
index 517b312..268dbe7 100644
--- a/test/common.js
+++ b/test/common.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
@@ -234,7 +234,10 @@ function waitForValue(url, key, value, options, callback) {
     assert.func(callback, 'callback');
 
     var client = options.client;
-    var timeout = 120;
+
+    // Set a longish timeout value to allow for slow tests that do eventually
+    // complete. For instance, concurrent test execution on older hardware.
+    var timeout = 10 * 60;
     var times = 0;
 
     if (options.timeout !== undefined) {
-- 
2.21.0

